src/composer.ts

```diff
@@ -137,19 +179,26 @@ export class AtomicTransactionComposer {
    * for the given method.
    */
   addMethodCall({
```

Comment:
Also, do we want to let users pass in their own foreign object arrays to the method call?

Reply:
We probably should, but I chose to ignore this in the cucumber tests and JS and Go SDK implementations in order to get to a release sooner.

I can make follow up issues for it, but since this is a less necessary feature, I don't think it should block a release.

Though if the Python or Java SDK have already implemented it, that's perfectly fine.